src/integration/testdata/prometheus_exporter.yml

```diff
@@ -28,4 +28,9 @@ service:
       exporters: [prometheus]
   telemetry:
     metrics:
-      address: "127.0.0.1:{{.MetricsPort}}"
+      readers:
+        - pull:
+            exporter:
+              prometheus:
+                host: "127.0.0.1"
+                port: {{.MetricsPort}}
```

Comment:
Please add back the new line at the end of the file

Reply:
fixed